test/java/seedu/addressbook/commands/CountCommandTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import org.junit.Assert;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+
+import java.util.HashSet;
+
+public class CountCommandTest {
+
+    AddressBook testAddressBook = new AddressBook();
+
+    @Test
+    public void testCountCommandOutput() {
+        try {
+            testAddressBook.addPerson(new Person(
+                    new Name("A"),
+                    new Phone("1", true),
+                    new Email("A@gmail.com", true),
+                    new Address("A", true),
+                    new HashSet<>())
+            );
+
+            testAddressBook.addPerson(new Person(
+                    new Name("B"),
+                    new Phone("2", true),
+                    new Email("B@gmail.com", true),
+                    new Address("B", true),
+                    new HashSet<>())
+            );
+
+            testAddressBook.addPerson(new Person(
+                    new Name("C"),
+                    new Phone("3", true),
+                    new Email("C@gmail.com", true),
+                    new Address("C", true),
+                    new HashSet<>())
+            );
+
+            CountCommand testCommand = new CountCommand();
+            testCommand.setData(testAddressBook,null);
+            CommandResult testResult = testCommand.execute();
+
+            Assert.assertEquals(CountCommand.MESSAGE_SUCCESS + 3, testResult.feedbackToUser);
```

Comment:
This is a good example for testing a success case for 3 people. How about adding tests for the case where the address book is empty?